code/modules/power/apc/apc.dm

```diff
@@ -488,6 +491,9 @@
 	if(issilicon(user))
 		var/mob/living/silicon/ai/AI = user
 		var/mob/living/silicon/robot/robot = user
+		if(malfai == "DVORAK") //I'm not making an actual ai core for this, you get a string.
```

Comment:
What happens if a player AI names themselves this?

Reply:
Nothing, as under normal circumstances, malfai is a reference to a silicon, not a string.